src/ExEss/Cms/Api/V8_Custom/Service/DataCleaner.php

```diff
@@ -10,6 +10,8 @@
  */
 class DataCleaner
 {
+    public const STREAM_REPLACEMENT = 'stream-not-logged';
```

Comment:
I'm don't see this being used somewhere else, are you planning something?

Reply:
nice spot, was planning to use it but forgot afterwards, will push another commit